tests/files/fpu_test.asm

```diff
@@ -0,0 +1,89 @@
+; Authored by finalpatch (GitHub)
+;   Source: https://gist.github.com/finalpatch/91ae4449875c3cf217327bfdcbeab0c5
+;   Tests: https://github.com/dosbox-staging/dosbox-staging/issues/2247
+;
+; tasm fpu
+; tlink fpu
```

Comment:
this is the assembler/linker instruction.  since you changed filename to fpu_test,  you need to update this as well.

Reply:
> this is the assembler/linker instruction. 

Yes, I just don't have a tasm or tlink (but I do have `fasm`).
If you don't have the binary handy, I guess I'll track these down later.

>  since you changed filename to fpu_test, you need to update this as well

Will do.

Reply:
In bound.